tests/test_proxies.py

```diff
@@ -13,8 +13,8 @@ def test_get_most_recent_block(self):
         api = Proxies(api_key=API_KEY)
         # currently raises an exception even though it should not, see:
         # https://github.com/corpetty/py-etherscan-api/issues/32
```

Comment:
We forgot to remove that comment

Reply:
Yes the two lines comments were with a `self.assertRaises(EmptyResponse)` to describe a bug. But the bug was fixed and assert removed in 06f7b0435d2546e3c25938bf3e11f0487b4d6a20, but the comment was left. So yes basically just removing that two comment is enough.
Also you can delete the `print`, I don't think we need `print` in unit tests. Plus I think we should avoid making actual network calls, but actual mock it, but that's a different story.

I'm not sure regarding the rainy scenario. with error code and message.

Reply:
True for the linter it's currently only tests folder, but that one is also failing. e.g. `tests/test_proxies.py:4:1: F401 'etherscan.client.EmptyResponse' imported but unused` https://github.com/corpetty/py-etherscan-api/blob/06f7b04/tests/test_proxies.py#L4 and other couple of more. So really not sure why Travis didn't catch these obvious ones.
They don't pop up when you run `tox` on your dev env?

Reply:
I just ran flake8?

Reply:
No I ran `tox` that rans `flake8` for you.
```
tox
GLOB sdist-make: /home/andre/workspace/py-etherscan-api/setup.py
pep8 inst-nodeps: /home/andre/workspace/py-etherscan-api/.tox/dist/py_etherscan_api-0.7.0.zip
pep8 installed: certifi==2018.4.16,chardet==3.0.4,flake8==3.5.0,idna==2.6,mccabe==0.6.1,py-etherscan-api==0.7.0,pycodestyle==2.3.1,pyflakes==1.6.0,requests==2.18.4,urllib3==1.22
pep8 runtests: PYTHONHASHSEED='913872812'
pep8 runtests: commands[0] | flake8 tests/
tests/test_proxies.py:4:1: F401 'etherscan.client.EmptyResponse' imported but unused
tests/test_proxies.py:20:1: W391 blank line at end of file
tests/test_proxies.py:20:1: W293 blank line contains whitespace
ERROR: InvocationError: '/home/andre/workspace/py-etherscan-api/.tox/pep8/bin/flake8 tests/'
py3 inst-nodeps: /home/andre/workspace/py-etherscan-api/.tox/dist/py_etherscan_api-0.7.0.zip
py3 installed: atomicwrites==1.1.5,attrs==18.1.0,certifi==2018.4.16,chardet==3.0.4,idna==2.6,more-itertools==4.2.0,pluggy==0.6.0,py==1.5.3,py-etherscan-api==0.7.0,pytest==3.6.2,requests==2.18.4,six==1.11.0,typing==3.6.4,urllib3==1.22
py3 runtests: PYTHONHASHSEED='913872812'
py3 runtests: commands[0] | python -m unittest discover --start-directory=tests/
5848317
/usr/lib/python3.6/unittest/case.py:605: ResourceWarning: unclosed <ssl.SSLSocket fd=3, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.1.130', 42938), raddr=('23.111.151.66', 443)>
  testMethod()
./usr/lib/python3.6/unittest/case.py:605: ResourceWarning: unclosed <ssl.SSLSocket fd=3, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.1.130', 42940), raddr=('23.111.151.66', 443)>
  testMethod()
./usr/lib/python3.6/unittest/case.py:605: ResourceWarning: unclosed <ssl.SSLSocket fd=3, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.1.130', 42942), raddr=('23.111.151.66', 443)>
  testMethod()
.
----------------------------------------------------------------------
Ran 3 tests in 2.913s

OK
_____________________________________________________________________________________________________________________________________ summary _____________________________________________________________________________________________________________________________________
ERROR:   pep8: commands failed
  py3: commands succeeded
```

Reply:
Okay just ran that. Got the same results. When should we switch flake8 to the whole directory?